content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -13,14 +14,22 @@ You need to balance compatibility and features:
 In particular, the LTS Release Line is based on slightly older releases to provide a more stable experience for conservative users.
 * *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin. 
 You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases. 
-Additionally, since features are sometimes moved from core into plugins, depending on a more recent Jenkins version will make your plugin's dependencies on what used to be core features explicit, otherwise your plugin will not work.
-* *Use at least the minimum version supported by the update center*, the supported update center versions can be found at link:https://updates.jenkins.io[updates.jenkins.io], at the time of writing (2020-03) this is 2.164.x.
+* *Do not use versions no longer supported by the update center*, which is currently anything older than PLACEHOLDER_OLDEST_WEEKLY for weekly releases, and PLACEHOLDER_OLDEST_LTS for LTS releases.
+* *Prefer ".1" LTS releases over weekly versions and later releases within an LTS line* for greater compatibility.
+  LTS releases with the same baseline usually don't have differences that would matter to plugin developers, and if they do, they may well behave differently on supposedly compatible weekly releases published shortly after the LTS baseline.
```

Comment:
Not sure I agree with this advice (the `*.1` part). Better to be testing the plugin against a core version that users are actually likely to have installed than some obsolete version with subtly different behavior. As for weeklies published after the LTS baseline, well yes but the same problem applies to `*.1` also.